Chessington.GameEngine/Pieces/Piece.cs

```diff
@@ -10,10 +11,26 @@ protected Piece(Player player)
         {
             Player = player;
             HasMoved = false;
+            CardinalDirections = new List<Tuple<int, int>>
+            {
+                new Tuple<int, int>(0, 1),
```

Comment:
Instead of using `Tuple` I'd recommend creating a custom class. That should make this more readable 🙂 

Also is there any reason why this needs to have a private set and be initialised here? They can be constants I believe as they should never change 🙂